action-api/src/test/java/com/powsybl/action/EqualsActionTest.java

```diff
@@ -0,0 +1,472 @@
+                .withRegulationMode(PhaseTapChanger.RegulationMode.CURRENT_LIMITER)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void ratioTapChangerRegulationAction() {
+        Action action1 = new RatioTapChangerRegulationActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTargetV(2.)
+                .build();
+        RatioTapChangerRegulationAction action2 = new RatioTapChangerRegulationActionBuilder().withId("id")
+                .withTransformerId("neId")
+                .withTargetV(2.)
+                .build();
+        Action action3 = new RatioTapChangerRegulationActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTargetV(2.)
+                .withSide(ThreeSides.ONE)
+                .build();
+        Action action4 = new PhaseTapChangerRegulationActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withRegulationValue(2.)
+                .build();
+        RatioTapChangerRegulationAction action5 = new RatioTapChangerRegulationActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTargetV(2.)
+                .withRegulating(true)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void ratioTapChangerTapPositionAction() {
+        Action action1 = new RatioTapChangerTapPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(false)
+                .build();
+        RatioTapChangerTapPositionAction action2 = new RatioTapChangerTapPositionActionBuilder().withId("id")
+                .withTransformerId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(false)
+                .build();
+        Action action3 = new RatioTapChangerTapPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(false)
+                .withSide(ThreeSides.ONE)
+                .build();
+        Action action4 = new PhaseTapChangerTapPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withTapPosition(2)
+                .withRelativeValue(false)
+                .build();
+        RatioTapChangerTapPositionAction action5 = new RatioTapChangerTapPositionActionBuilder().withId("id")
```

Comment:
To fully cover the `equals()` method, could you add an action with a change on the following attribute:
- tapPosition